### PR TITLE
Add openshift-pipelines fips assessment task(s)

### LINKED ISSUE #16: Add OpenShift-Pipelines to fips-assessments tasks
Add Ansible tasks that can perform the following checks: https://github.com/opdev/fips-assessments/blob/main/openshift-pipelines/check.txt

opcap-ansible integration will be treated on https://github.com/opdev/opcap-ansible/issues/22

### PR DESCRIPTION
**Pull request by mrhillsman**
Resolves #16

### CODE REVIEW

**Review by bcrochet — CHANGES_REQUESTED** [review]
Couple of requests, but overall looking good.

File: openshift-pipelines/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+          value: image-registry.openshift-image-registry.svc:5000/$(context.pipelineRun.namespace)/pipelines-vote-ui
+        workspaces:
+        - name: shared-workspace
+          volumeClaimTemplate:
+            spec:
+              accessModes:
+                - ReadWriteOnce
+              resources:
+                requests:
+                  storage: 500Mi
```
[the remark this answers is not in the document]
Comment (mrhillsman) [reply openshift-pipelines/tasks/main.yml]: Yes

File: openshift-pipelines/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+# Ansible create a new OpenShift/Kubernetes namespace pipelines-tutorial
+- name: Create the pipelines-tutorial namespace
+  community.kubernetes.k8s:
+    state: present
+    api_version: v1
+    kind: Namespace
+    name: pipelines-tutorial
+
+# Check if ServiceAccount pipeline exists in pipelines-tutorial namespace; if not create it and then verify it exists
+- name: Check if ServiceAccount pipeline exists in pipelines-tutorial namespace
```
[the remark this answers is not in the document]
Comment (mrhillsman) [reply openshift-pipelines/tasks/main.yml]: Done

File: openshift-pipelines/tasks/main.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (mrhillsman) [reply openshift-pipelines/tasks/main.yml]: Done

**Review by bcrochet — COMMENTED** [review]
Just a question about the definitions.

**Comment by mrhillsman** [comment]
Just leaving a note that I've see the pipelinerun fail due to not pulling down GitHub repo so need to do some refactoring for the pipelineruns at a minimum.

File: openshift-pipelines/tasks/main.yml [hunk 3]
```diff
@@ -0,0 +1,266 @@
...
+                resources:
+                  requests:
+                    storage: 100Mi
+    wait: true
+    wait_timeout: 90
+    wait_condition:
+      type: Succeeded
+      status: "True"
+      reason: Succeeded
+
```
Comment (bcrochet) [inline openshift-pipelines/tasks/main.yml:253]: Looks like this still needs to report the result back?

File: openshift-pipelines/tasks/main.yml [hunk 4]
```diff
@@ -0,0 +1,266 @@
...
+# Remove the pipelines-tutorial namespace; will remove all the resources created in the namespace
+- name: Remove the pipelines-tutorial namespace
+  kubernetes.core.k8s:
+    state: absent
+    api_version: v1
+    kind: Namespace
+    name: pipelines-tutorial
+    wait: true
+    wait_timeout: 60
+    wait_condition:
```
Comment (bcrochet) [inline openshift-pipelines/tasks/main.yml:263]: This isn't strictly necessary. For state: absent, and wait: true it automatically does the right thing, and waits until it is actually deleted.

**Review by bcrochet — CHANGES_REQUESTED** [review]
Just a couple of comments.

**Review by bcrochet — APPROVED** [review]
/lgtm

**Pull request merged**